packages/discord.js/typings/index.d.ts

```diff
@@ -2453,6 +2453,16 @@ export class Webhook extends WebhookMixin() {
   public sourceChannel: NewsChannel | APIPartialChannel | null;
   public token: string | null;
   public type: WebhookType;
+  public applicationId: Snowflake | null;
+  public isUserCreated(): this is this & {
+    applicationId: null;
+    owner: User | APIUser;
+    token: null;
+  };
+  public isApplicationCreated(): this is this & {
+    applicationId: Snowflake;
+    owner: null;
+  };
   public isIncoming(): this is this & { token: string };
   public isChannelFollower(): this is this & {
```

Comment:
Can't the type of webhook also be narrowed here?

Reply:
Narrowed the types for all of the checks.